lib/utils.js

```diff
@@ -34,7 +34,7 @@ function isPromise(obj) {
 }
 
 function isFalsy(val) {
-  return !val ||
+  return val === false ||
```

Comment:
I think my comment was about `!val || val === false` being redundant. But I didn't realise `!val` would cause handlers without a `return` statement to block the save hook, which doesn't happen in v1.